GOVERNANCE.md

```diff
@@ -9,49 +9,72 @@ In order to provide equitable rights to all Working Group members,
 the following process will be followed:
 
 * Official WG meetings will be announced at least a week in advance.
-* Proposed changes to any document will be done via a Pull Request (PR).
-* PRs will be reviewed during official WG meetings.
+* Proposed changes to any document will be done via a Pull Request (PR), 
+  see below for policy on review and acceptance of PRs.
 * During meetings, priority will be given to PRs that appear to be ready for
   a vote over those that appear to require discussions.
-* PRs should not be merged if they have had substantial changes made within
-  two days of the meeting.
-  Rebases, typo fixes, etc. do not count as substantial.
-  Note, administrivia PRs that do not materially modify WG output documents
-  may be processed by WG admins as needed.
-* Resolving PRs ("merging" or "closing with no action") will be done as a
-  result of a motion made during a WG meeting, and approved.
-* Reopening a PR can be done if new information is made available, and a
-  motion to do so is approved.
 * Any motion that does not have "unanimous consent" will result in a formal
   vote. See [Voting](#voting).
 
+## Working Group Membership
+
+Anyone is welcome to attend Working Group meetings and invited to review 
+PRs, whether or not they have declared formal membership in the Working Group.
+
+We encourage everyone to document your participation in the [member list](), 
+then people new to the group can learn about other folks who are involved.
+Each member identifies their affiliation with an organization, company or
```

Comment:
(From offline conversation, because I was confused)

```
Each member (company, organization, or independent project) which
wishes to be considered for membership should add their name to the
[member list]() with the following information:

[existing template]
```